defaults/main.yml

```diff
@@ -269,13 +409,15 @@ owncloud__variant_name_map:
 #
 # Supported releases:
 #
-# * ownCloud ``8.1``
-# * ownCloud ``8.2``
 # * ownCloud ``9.0`` (main supported version and used in production by the role maintainers)
 # * ownCloud ``9.1`` (setup should work but not yet well tested nor used in production by the role maintainers)
 #
+# * Nextcloud ``11.0`` (experimental support on Debian stretch, refer to `issue 45 <https://github.com/debops/ansible-owncloud/issues/45>`_ for details)
+#
 # Refer to the `ownCloud Maintenance and Release Schedule <https://github.com/owncloud/core/wiki/Maintenance-and-Release-Schedule>`_
 # and the `package index <https://download.owncloud.org/download/repositories/>`_ for more details.
+#
+# For Nextcloud refer to the `Nextcloud Server Changelog <https://nextcloud.com/changelog/>`_.
 owncloud__release: '9.0'
```

Comment:
To make things easier, perhaps this variable could be conditionally set based on `owncloud__variant`? Something like:
```yaml
owncloud__release: '{{ "9.0"
                       if (owncloud__variant == "owncloud")
                       else "11.0" }}'
```